### PR TITLE
Answered question to when to use node

### PR DESCRIPTION
**Pull request by daniel-vera-g**
Answered the question to where and when to use nodejs.

### CODE REVIEW

File: node.md [hunk 1]
```diff
@@ -27,8 +27,12 @@ _Waiting for response_
 <details>
 <summary>Does Node replace Java or Python?</summary>
   
-If you know the answer to this question, please submit a pull request with the answer.
+Node, Java and Python have different strengths.
 
-</details>
+With Node you get a Robust technology stack, it is fast due to it's event based character, you can use it very good for scaling(Microservices) and it has a rich ecosystem. On the Other hand, due to his non-blocking I/O it is not suited for computation-heavy tasks like machine learning.
```
Comment (emmabostian) [inline node.md]:
```suggestion
With Node you get a Robust technology stack, it is fast due to it's event based character, you can use it very good for scaling(Microservices) and it has a rich ecosystem. On the other hand, due to his non-blocking I/O it is not suited for computation-heavy tasks like machine learning.
```

**Review by emmabostian — CHANGES_REQUESTED** [review]
Just a minor capitalization nitpick! Let me know once it's fixed!

**Comment by daniel-vera-g** [comment]
Used your suggestion. Should be updated.

**Review by emmabostian — APPROVED** [review]
Thanks!

**Pull request merged**